networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/hobbits/SocketHandler.java

```diff
@@ -42,53 +43,67 @@
 import tech.pegasys.artemis.networking.p2p.hobbits.rpc.RPCCodec;
 import tech.pegasys.artemis.networking.p2p.hobbits.rpc.RPCMessage;
 import tech.pegasys.artemis.networking.p2p.hobbits.rpc.RPCMethod;
+import tech.pegasys.artemis.networking.p2p.hobbits.rpc.RequestAttestationMessage;
 import tech.pegasys.artemis.networking.p2p.hobbits.rpc.RequestBlocksMessage;
 import tech.pegasys.artemis.storage.ChainStorageClient;
 import tech.pegasys.artemis.util.alogger.ALogger;
 
 /** TCP persistent connection handler for hobbits messages. */
-public final class HobbitsSocketHandler {
-  private static final ALogger LOG = new ALogger(HobbitsSocketHandler.class.getName());
-  private final EventBus eventBus;
-  private final String userAgent;
-  private final Peer peer;
-  private final ChainStorageClient store;
-  private final Set<Long> pendingResponses = new HashSet<>();
-  private final AtomicBoolean status = new AtomicBoolean(true);
-  private final Consumer<Bytes> messageSender;
-  private final Runnable handlerTermination;
-  private final State p2pState;
+public abstract class SocketHandler {
```

Comment:
You might wanna change this file's name.

Reply:
changing now